cluster/gce/gci/configure.sh

```diff
@@ -55,6 +55,17 @@ Node instance:
 EOF
 }
 
+function validate-python {
+  local ver=$(python -c"import sys; print(sys.version_info.major)")
```

Comment:
since python and python3 is separated command, this will always return 2 and python3 will return 3. so this logic need updated to handle those ? it probably just ensure python3 is installed.

do we really need this validation btw, should GCE OS image already has the desired version installed ?